test/passing/long_let_in.ml

```diff
@@ -0,0 +1,36 @@
+let () =
```

Comment:
It isn't discoverable, sorry, but we are trying to move away from using the `.opts` files unless necessary. So in cases like this it is easier to work with to add an attribute at the toplevel of the test file. It can also be helpful to add another attribute after the tests with the option set the other way and duplicate (some of) the tests to show the effect.

```suggestion
[@@@ocamlformat "end-of-line-in=true"]

let () =
```

Reply:
Done